litgpt/api.py

```diff
@@ -196,7 +197,9 @@ def generate(
         Arguments:
             model: The model to use.
             prompt: The prompt string to use for generating the samples.
-            max_new_tokens: The maximum number of tokens to generate.
+            max_new_tokens: The maximum number of new tokens to return.
+            max_seq_length: The size of kvcache to use. If 'dynamic', the kvcache size will be
+                sized to the max returned tokens, up to 'max_model_supported'.
```

Comment:
Just adding some wording to explain why one would do one over the other.

```suggestion
                resized to the max returned tokens, up to 'max_model_supported', to lower memory requirements. Choose a fixed size by providing an integer < 'max_model_supported' or 'max_model_supported' to avoid dynamic resizing, for example, when compiling the model.
```